test/jdk/tools/jlink/JLinkReproducibleTest.java

```diff
@@ -37,6 +37,9 @@
  * @run driver JLinkReproducibleTest
  */
 public class JLinkReproducibleTest {
+
+    static final String TOOL_VM_OPTIONS = System.getProperty("test.tool.vm.opts", "");
```

Comment:
Nit - this can be made `private`

Reply:
Thanks for the review. The variable has made to private.